templates/vue/src/components/Lightbox/media/WpPostMedia.vue

```diff
@@ -61,6 +61,7 @@ export default {
   text-align: left;
 
   &-title {
+    color: #111;
```

Comment:
Again, not sure if we need this on a child element - can we keep it on the parent elements instead? Is there an overwrite somewhere that should be removed instead?

Reply:
Yep, moved to parent